dotnet-cognito-auth/donet-api-gateway-auth/ApiHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace donet_api_gateway_auth
+{
+    public static class ApiHelper
+    {
+        private static string GenerateOauthToken()
+        {
+            //TODO: pull this from env vars
+            string consumerKey = "5mulvgjjlbbm4cb8p7s1d6li8i";
```

Comment:
Client id and secret should not be committed, please add them to  environment variables